kaggle_environments/envs/mab/agents.py

```diff
@@ -0,0 +1,10 @@
+import random
+
+
+def random_agent(observation, configuration):
```

Comment:
@pculliton Ideas for other agents to add in here?

Reply:
@harrisse Yep! I've got a UCB agent sitting around that might be a useful bar as well. I'll test it some more.

Reply:
Maybe one that pulls `stepNum % banditCount`?